pkg/scheduler/statefulset/scheduler.go

```diff
@@ -707,3 +710,15 @@ func (s *StatefulSetScheduler) makeZeroPlacements(vpod scheduler.VPod, placement
 	// free capacity when there are existing placements for a vpod
 	s.reservePlacements(vpod, newPlacements)
 }
+
+// newNotEnoughPodReplicas returns an error explaining what is the problem, what are the actions we're taking
+// to try to fix it (retry), wrapping a controller.requeueKeyError which signals to ReconcileKind to requeue the
+// object after a given delay.
+func (s *StatefulSetScheduler) newNotEnoughPodReplicas(left int32) error {
```

Comment:
Do we need the `new` in the `newNotEnoughPodReplicas` func name?

Reply:
No, we don't, should I remove it?

Reply:
Since it has no meaning here, yes please

Reply:
the meaning was "create a _new_ NotEnoughPodReplicas error" but 'll remove it anyway since it's not strictly needed to convey what is doing

Reply:
right.. figured it was for new err and/or new requeue